style.css

```diff
@@ -0,0 +1,21 @@
+#createBook {
+  position: fixed;
+  bottom: 0;
+  right: 0;
+  padding: 1rem 1rem;
+  background-color: white;
+
+  #form {
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+
+    #input {
+      padding: 0.5rem;
+    }
+
+    .btn {
+      padding: 0.5rem;
+    }
+  }
+}
```

Comment:
- [ ] It seems like you are trying to use `SASS` in your CSS code and maybe because of it your `Output` or design is not matching with the given `design` provided by the **Microverse**. It is very important to follow the minimum requirement of the project. Kindly adhere to the project requirement and have a look at the below-attached image and table.

### Design Comparison
<table>
<tr>
<th>Figma Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

<img width="348" alt="awesome_books_basic_ui" src="https://github.com/Wils-FOURR-speed/Awesome-Books/assets/66476329/e193168e-094b-4fdf-ac37-c15645343d87">

</td>
<td>

![Screenshot from 2023-07-04 15-40-07](https://github.com/Wils-FOURR-speed/Awesome-Books/assets/66476329/ab9cd104-96ec-4597-9165-0f10965a7d20)
</td>
</tr>
</table>

Reply:
Bro this is not SASS. This is nested CSS . let me remove CSS style

Reply:
@sheezu900 Please use only plain CSS, not any other pre-processor or library for the first module.